File: coding_meetup_5.js
function countLanguages(list) {
    // thank you for checking out the Coding Meetup kata :)
    //we create an empty object
    let countObject={}
     list.forEach((dev)=>{
       if(dev.language) countObject[`${dev.language}`]=0})
    //we get the languages present
    let languagesPresent=Object.keys(countObject)
  for(j=0;j<languagesPresent.length;j++){
    for(i=j;i<list.length;i++){
      if(languagesPresent[j]===list[i].language){
        console.log(list[i].language)
        countObject[list[i].language]++
      }
    }
  }
    console.log(countObject)
    return countObject
  }     

File: coding_meetup_5.py
def count_languages(lst):
    count_lang_dict = {}
    for dev in lst:
        if dev['language'] not in count_lang_dict:
            count_lang_dict[dev['language']] = 1
        else:
            count_lang_dict[dev['language']] += 1
    return count_lang_dict